addPost.php

```diff
@@ -0,0 +1,101 @@
+<?php
+    require_once 'src/Models/BlogModel.php';
+    require_once 'src/connectToDb.php';
+
+    session_start();
+    $db = connectToDb();
+
+    if (!isset($_SESSION['userid'])) {
+        header('Location: src/login.php');
+    } else {
+        $authorid = $_SESSION['userid'];
+    }
+
+    $title = '';
+    $content = '';
+    $titleError = '';
+    $contentError = '';
+    $successMessage = '';
+    $model = new BlogModel($db);
+    $titleCheck=false;
+    $contentCheck=false;
+
+    function cleanUpInput($data) {
+        $data = trim($data);
+        return htmlspecialchars($data);
+    }
+
+    if ($_SERVER["REQUEST_METHOD"] == "POST") {
```

Comment:
same as before, you need to check the title and content in the post request, instead of checking if it is a post request

Reply:
done